### DISCUSSION TITLE
Need Clarification on Upcoming Atlantis Template Update – Tailwind Support and PrimeVue 4

### DISCUSSION THREAD

**Discussion by M-Atef-ZC** [primefaces/community#2579] [PrimeVue Templates] [answered]
Hello ,I recently purchased an extended license for the Atlantis template, and I’m encountering issues because it currently works with PrimeFlex, while my project requires Tailwind. I believe the steps in the PrimeFlex to Tailwind Conversion Guide (https://primevue.org/guides/primeflex/) didn’t work for me because the template is built with PrimeVue 3.

I’ve noticed that there’s an upcoming update for your templates planned for early September. I have a few questions regarding this update:

1. Will the upcoming update for the Atlantis template include:
- An upgrade to PrimeVue 4?
- A switch from PrimeFlex to Tailwind?

2. Could you clarify what is meant by "early September" for the release of the update? I need to decide whether to wait for this update or proceed with implementing the current version, as I’m running on a tight schedule. I want to avoid duplicating work if possible.

I appreciate any guidance or information you can provide. Thank you!

**Comment by cantin-f** [comment]
I'm not from Primetek but from what I've read :
1- Yes and Yes
2- Early septembrer is the last schedule update I've seen

**Reply by M-Atef-ZC** [reply]
thank you for your respond

**Comment by tugcekucukoglu** [comment, marked as answer]
The new version of Atlantis with Tailwind can be downloaded from the PrimeStore.
